adam/perception/__init__.py

```diff
@@ -77,6 +77,15 @@ class PerceptualRepresentationFrame(ABC):
     """
 
 
+@attrs(slots=True, frozen=True, repr=False)
+class VisualPerceptionFrame(PerceptualRepresentationFrame):
+    r"""
+    A static snapshot of a visually processed representation of an image.
+    This is the default perceptual representation for phase 3 phase of the ADAM project.
+    """
+    perception_yaml: Dict[Any, Any] = attrib(validator=instance_of(Dict))
```

Comment:
Prefer `Mapping` or `MutableMapping` as types opposed to `Dic` as the former is more generic.